src/components/ThreeDEditor.jsx

```diff
@@ -22,14 +22,13 @@ import LooksIcon from "@mui/icons-material/Looks";
 import Menu from "@mui/icons-material/Menu";
 import PictureInPicture from "@mui/icons-material/PictureInPicture";
 import PowerSettingsNew from "@mui/icons-material/PowerSettingsNew";
-// import RemoveRedEye from "@mui/icons-material/RemoveRedEye";
+import RemoveRedEye from "@mui/icons-material/RemoveRedEye";
 import Replay from "@mui/icons-material/Replay";
 import Spellcheck from "@mui/icons-material/Spellcheck";
 import SquareFootIcon from "@mui/icons-material/SquareFoot";
 import SwitchCamera from "@mui/icons-material/SwitchCamera";
 import ThreeDRotation from "@mui/icons-material/ThreeDRotation";
-import { createTheme, ThemeProvider, Tooltip } from "@mui/material";
-// import { styled } from "@mui/system";;
+import { ButtonGroup, createTheme, ThemeProvider, Tooltip } from "@mui/material";
```

Comment:
Do not import the whole lib, import individual components eg. `import from "@mui/material/Tooltip";`